src/components/admin/services/timer/controls.tsx

```diff
@@ -1,30 +1,13 @@
-import { Trash2, Pen, Check } from "lucide-react";
+import { Trash2 } from "lucide-react";
 
 type props = {
-  edit: boolean;
-  setEdit: (value: boolean) => void;
   onRemove: () => void;
 };
 
-const Controls = ({ edit, setEdit, onRemove }: props) => {
+const Controls = ({ onRemove }: props) => {
```

Comment:
dont need a component just to call a single div / icon, lets combine into the parent component